Sources/YBottomSheet/Animation/BottomSheetPresentAnimator.swift

```diff
@@ -21,11 +21,13 @@ class BottomSheetPresentAnimator: BottomSheetAnimator {
         sheet.view.layoutSubviews()
         sheet.dimmerView.alpha = 0
         
-        let toFinalFrame = transitionContext.finalFrame(for: toViewController)
-        var sheetFrame = sheet.sheetView.frame
-        sheetFrame.origin.y = toFinalFrame.maxY + (sheet.appearance.elevation?.extent.top ?? 0)
-        sheet.sheetView.frame = sheetFrame
-        sheet.view.setNeedsLayout()
+        if !isReduceMotionEnabled {
+            let toFinalFrame = transitionContext.finalFrame(for: toViewController)
+            var sheetFrame = sheet.sheetView.frame
+            sheetFrame.origin.y = toFinalFrame.maxY + (sheet.appearance.elevation?.extent.top ?? 0)
+            sheet.sheetView.frame = sheetFrame
+            sheet.view.setNeedsLayout()
+        }
```

Comment:
Where are you fading the view in?
Should be if isReduceMotionEnabled then fade in else do the frame translation.